src/index.html

```diff
@@ -2,12 +2,58 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta
+      name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+    >
     <title>Product cards</title>
-    <link rel="stylesheet" href="./style.css">
+    <link
+      rel="stylesheet"
+      href="./style.css"
+    >
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="stars stars--0">
+      <div class="grey-star"></div>
+      <div class="grey-star"></div>
+      <div class="grey-star"></div>
+      <div class="grey-star"></div>
+      <div class="grey-star"></div>
+    </div>
+    <div class="stars stars--1">
+      <div class="yellow-star"></div>
+      <div class="grey-star"></div>
+      <div class="grey-star"></div>
+      <div class="grey-star"></div>
+      <div class="grey-star"></div>
+    </div>
+    <div class="stars stars--2">
+      <div class="yellow-star"></div>
+      <div class="yellow-star"></div>
+      <div class="grey-star"></div>
+      <div class="grey-star"></div>
+      <div class="grey-star"></div>
```

Comment:
```suggestion
      <div class="stars__star"></div>
      <div class="stars__star"></div>
      <div class="stars__star"></div>
      <div class="stars__star"></div>
      <div class="stars__star"></div>
```
Тут должны быть вот такие классы, а определять какой background должен быть у звездочки можно по ее порядку, вместо специального класса